Makefile

```diff
@@ -27,9 +27,11 @@ proto:
 	done
 
 test: testdeps
+	go test -i google.golang.org/grpc/...
```

Comment:
I did a bit investigation. It seems due to some google specific golang setup. But at very least, this will make me unable to "make" when I work. :)